spanner/client.go

```diff
@@ -147,24 +152,33 @@ type ClientConfig struct {
 	// database by this client.
 	DatabaseRole string
 
+	// RouteToLeader specifies if all the requests of type read-write and PDML
+	// need to be routed to the leader region.
+	//
+	// Default: true
+	RouteToLeader bool
```

Comment:
I think that we need to invert this to `DisableRouteToLeader` with a default false value. The problem now is that anyone who is currently using `NewClientWithConfig(ClientConfig{ .... })` will create a config with `RouteToLeader=false`, unless they change their code after this change. We want this option to be enabled by default, but in the current setup, it won't be for anyone who does not use the config-less `NewClient(...)` method.